Sofya_Huliuk_Application/Program.cs

```diff
@@ -294,8 +294,161 @@ public void Task_9()
         uint d = (uint)b;
     }
 
+    public void Task_10()
+    {
+
+    }
+
 
 
 
 
 }
+
+class Homework_3
+{
+    public void Task_1()
+    {
+        while (true)
+        {
+            Console.WriteLine("Enter the integer:");
+            string input = Console.ReadLine();
+            bool result = int.TryParse(input, out var num);
+            if (!result)
+
+            {
+                Console.WriteLine("incorrect value, please try again");
+                continue;
+            }
+
+            if (num % 10 == 0)
+            {
+                Console.WriteLine("tutti-frutti");
+
+                continue;
+            }
+
+            if (num % 2 == 0)
+            {
+                Console.WriteLine("tutti");
+
+                continue;
+            }
+
+            if (num % 5 == 0)
+            {
+                Console.WriteLine("frutti");
+
+                continue;
+            }
+        }
+
+    }
+    public void Task_2()
+    {
+        Console.WriteLine("Enter the first number:");
+        string input1 = Console.ReadLine();
+        bool result1 = int.TryParse(input1, out var firstNum);
+        if (!result1)
+        {
+            Console.WriteLine("incorrect value, please try again");
+        }
+
+        Console.WriteLine("enter the second number, it must be greater than the first:");
+        string input2 = Console.ReadLine();
+        bool result2 = int.TryParse(input2, out var secondNum);
+        if (!result2)
+        {
+            Console.WriteLine("incorrect value, please try again");
+        }
+        if (secondNum <= firstNum)
+        {
+            Console.WriteLine("the second number must be greater than the first, please try again");
+        }
+
+
+        for (int value = firstNum; value <= secondNum; value++)
+        {
+            if (value % 10 == 0)
+            {
+                Console.WriteLine("tutti-frutti");
+
+                continue;
+            }
+
+            if (value % 2 == 0)
+            {
+                Console.WriteLine("tutti");
+
+                continue;
+            }
+
+            if (value % 5 == 0)
+            {
+                Console.WriteLine("frutti");
+
+                continue;
+            }
+
+            Console.WriteLine($"Number {value} can’t be divided on 2 or 5");
```

Comment:
would better to move this to else construction

Reply:
why is it better to use else construction?